### PR TITLE
Add player for video page, fix layout

### PR DESCRIPTION
**Pull request by andemerie**

### CODE REVIEW

**Comment by Drapegnik** [comment]
[Video article page markup](https://trello.com/c/iSx79dwg/137-video-article-page-markup)

**Comment by UladBohdan** [comment]
Now handling `videoUrl/videoId` in accordance with Backend in https://github.com/babajka/babajka-backend/pull/33

Tested:
* Associating youtube video with video articles
* Updating video of article
* Updating article type from `video` to `text` and vice versa

**Review by Drapegnik — DISMISSED** [review]

**Comment by UladBohdan** [comment]
[to comment 'Read more' section on Frontend](https://trello.com/c/dhI4eYnj/239-to-comment-read-more-section-on-frontend)

**Comment by UladBohdan** [comment]
[to make date on unpublished tiles responsive](https://trello.com/c/Fgffqe4R/227-to-make-date-on-unpublished-tiles-responsive)

**Comment by UladBohdan** [comment]
[Diary: make font of author's name less; open modal window not only by triggering 'read' button](https://trello.com/c/rq2M6L1M/245-diary-make-font-of-authors-name-less-open-modal-window-not-only-by-triggering-read-button)

**Comment by UladBohdan** [comment]
ok so how about merging this?

**Comment by UladBohdan** [comment]
I'm gonna fix Google Analytics here. And then it can be merged.

**Comment by UladBohdan** [comment]
[Google Analytics does not work neither on staging nor on production](https://trello.com/c/cB7vBEn0/246-google-analytics-does-not-work-neither-on-staging-nor-on-production)

File: components/specials/Diary.js [hunk 1]
```diff
@@ -98,8 +97,9 @@ class Diary extends Component {
     return (
       <div className="tile is-parent">
         <div className="card tile is-child diary-wrapper">
-          <article className="diary-article">
```
Comment (Drapegnik) [inline components/specials/Diary.js:101]: use `Clickable`

**Pull request merged**